lib/ansible/plugins/connection/network_cli.py

```diff
@@ -134,19 +136,30 @@ def put_file(self, in_path, out_path):
     def fetch_file(self, in_path, out_path):
         return self._local.fetch_file(in_path, out_path)
 
-    def update_play_context(self, play_context):
+    def update_play_context(self, pc_data):
```

Comment:
should this perhaps be part of `ConnectionBase` as an noop method?

Reply:
I'm not sure this makes sense as a method for any other connection plugins, at least not while the method expects a unicode string of a pickled PlayContext object, rather than the object itself.